velox/common/encode/Base64.cpp

```diff
@@ -163,10 +161,10 @@ std::string Base64::encodeImpl(
     const T& input,
     const Charset& charset,
     bool includePadding) {
-  size_t encodedSize = calculateEncodedSize(input.size(), includePadding);
+  const size_t encodedSize{calculateEncodedSize(input.size(), includePadding)};
```

Comment:
const is not meaningful for scalar values.

Reply:
updated the code